src/whylogs/app/session.py

```diff
@@ -76,6 +82,8 @@ def logger(
         profile_full_dataset: bool = False,
         with_rotation_time: str = None,
         cache_size: int = 1,
+        target_labels: List[str] = None,
+        model_name: str = None
```

Comment:
Is this one used anywhere? I assume `dataset_name` == `model_name` at the moment (Model data is a special case of dataset data)

Reply:
i removed `model_name` for now, will do it until we figure out the overall datastrtucture